dependencies.yaml

```diff
@@ -0,0 +1,185 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.5"]
+      arch: [x86_64]
+    includes:
+      - build
+      - cudatoolkit
+      - develop
+      - run
+      - test
+      - doc
+      - notebook
+  test:
+    output: none
+    includes:
+      - cudatoolkit
+      - test
+channels:
+  - rapidsai
+  - conda-forge
+# Not sure whether we need these:
```

Comment:
@galipremsagar Can I get your insight on this? Do we (always/sometimes/never) use the `dask/label/dev` channel?

Reply:
`dask/label/dev` is only needed during development through code-freeze. And `dask/label/dev` has to be dropped just like we drop `rapidsai-nightly` during release runs.

Reply:
@ajschmidt8 Same question for `rapidsai-nightly` -- when (if ever) do we need `rapidsai-nightly`?

I don't think we need the `nvidia` channel for cudf, but I think we will need it for cuML and cuGraph.